### PR TITLE
docs: restructure #2788

### PR DESCRIPTION
**Pull request by AnaisUrlichs**
Restructuring the docs as outlined #2788

### CODE REVIEW

File: docs/docs/deep-dive/override-os-files.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ Tracee will show you collected information about the running Linux OS with the
 `--log debug` argument:
 
 ```
-$ sudo ./dist/tracee-ebpf --log debug --filter uid=1000 --filter pid=new --filter event=execve
```
Comment (itaysk) [inline docs/docs/deep-dive/override-os-files.md:18]: we said this file would move to installation

File: docs/docs/events/events.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
+# Available Rules
```
Comment (itaysk) [inline docs/docs/events/events.md:1]: this should move to built-in events

File: docs/tutorials/setup-development-machine-with-vagrant.md [hunk 3]
```diff
@@ -105,7 +105,7 @@ from GitHub.
 
 ## Build and Run Tracee-eBPF and Tracee-Rules
```
Comment (itaysk) [inline docs/tutorials/setup-development-machine-with-vagrant.md:106]: we said this would move to contributing

File: docs/docs/events/index.md [hunk 4]
```diff
@@ -0,0 +1,165 @@
+# Getting Started with Detecting
```
Comment (itaysk) [inline docs/docs/events/index.md:1]: need to move events from contributing here

**Review by itaysk — COMMENTED** [review]
some gaps compared to #2788 (maybe more)

File: docs/tutorials/setup-development-machine-with-vagrant.md (hunk 3 above)
Comment (AnaisUrlichs) [reply docs/tutorials/setup-development-machine-with-vagrant.md:106]: yes you are right

File: docs/docs/events/index.md (hunk 4 above)
Comment (AnaisUrlichs) [reply docs/docs/events/index.md:1]: done

File: docs/tutorials/setup-development-machine-with-vagrant.md (hunk 3 above)
Comment (AnaisUrlichs) [reply docs/tutorials/setup-development-machine-with-vagrant.md:106]: done

File: docs/docs/events/events.md (hunk 2 above)
Comment (AnaisUrlichs) [reply docs/docs/events/events.md:1]: done

File: docs/docs/deep-dive/override-os-files.md (hunk 1 above)
Comment (AnaisUrlichs) [reply docs/docs/deep-dive/override-os-files.md:18]: done

**Comment by itaysk** [comment]
I took the "everything as event" tutorial and made it the k8s quickstart also heavily edited it. I've removed the part about editing tracee config, since it doesn't feel like a "quick" start with it, and also, with higher-level events it would be included default

**Comment by AnaisUrlichs** [comment]
@josedonizetti looks good to me

**Review by josedonizetti — APPROVED** [review]

**Pull request merged**